pytest_rabbitmq/factories/process.py

```diff
@@ -51,11 +51,12 @@ def get_conf_option(option: str) -> Any:
 
     port = get_conf_option("port")
     distribution_port = get_conf_option("distribution_port")
+    logsdir = get_config_option("logsdir")
```

Comment:
get_config_option

Reply:
Ah crap, managed to mess up a two line diff, sorry >_> thanks!

Reply:
Fixed in repush :)